src/test/cli/rbd/help.t

```diff
@@ -5,132 +5,145 @@
-                                            mirroring configuration.
-      mirror pool peer add                  Add a mirroring peer to a pool.
-      mirror pool peer remove               Remove a mirroring peer from a pool.
-      mirror pool peer set                  Update mirroring peer settings.
-      mirror pool promote                   Promote all non-primary images in the
-                                            pool.
-      mirror pool status                    Show status for all mirrored images
-                                            in the pool.
-      namespace create                      Create an RBD image namespace.
-      namespace list (namespace ls)         List RBD image namespaces.
-      namespace remove (namespace rm)       Remove an RBD image namespace.
-      object-map check                      Verify the object map is correct.
-      object-map rebuild                    Rebuild an invalid object map.
-      pool init                             Initialize pool for use by RBD.
-      remove (rm)                           Delete an image.
-      rename (mv)                           Rename image within pool.
-      resize                                Resize (expand or shrink) image.
-      snap create (snap add)                Create a snapshot.
-      snap limit clear                      Remove snapshot limit.
-      snap limit set                        Limit the number of snapshots.
-      snap list (snap ls)                   Dump list of image snapshots.
-      snap protect                          Prevent a snapshot from being deleted.
-      snap purge                            Delete all unprotected snapshots.
-      snap remove (snap rm)                 Delete a snapshot.
-      snap rename                           Rename a snapshot.
-      snap rollback (snap revert)           Rollback image to snapshot.
-      snap unprotect                        Allow a snapshot to be deleted.
-      status                                Show the status of this image.
-      trash list (trash ls)                 List trash images.
-      trash move (trash mv)                 Move an image to the trash.
-      trash purge                           Remove all expired images from trash.
-      trash remove (trash rm)               Remove an image from trash.
-      trash restore                         Restore an image from trash.
-      watch                                 Watch events on image.
+      bench                                   Simple benchmark.
+      children                                Display children of snapshot.
+      clone                                   Clone a snapshot into a CoW child
+                                              image.
+      config global get                       Get a global-level configuration
+                                              override.
+      config global list (config global ls)   List global-level configuration
```

Comment:
I am wondering if it's better to only list the last word in parenthesis to avoid these lengthy lines --- or perhaps something like "config global list (... ls)"